source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -80,22 +80,35 @@ Network::FilterStatus Filter::onAccept(Network::ListenerFilterCallbacks& cb) {
 
 void Filter::onRead() {
   try {
```

Comment:
Is this try/catch still needed?

Reply:
Yeah, at a minimum we still need it because we are calling https://github.com/envoyproxy/envoy/blob/116150e8acd2db1449e35097ecfe379a8cbac6d1/source/common/network/utility.cc#L159 (which throws) from `parseV1Header`.

Reply:
Thanks! Could we reduce the scope to just around this function? (with the hopes that it can be refactored later)